libs/block-editor/src/lib/plugins/dot-bubble-menu.plugin.ts

```diff
@@ -17,12 +17,43 @@ import {
 } from '@dotcms/block-editor';
 
 export const DotBubbleMenuPlugin = (options: DotBubbleMenuPluginProps) => {
+    const component = options.component.instance;
+
     return new Plugin({
         key:
             typeof options.pluginKey === 'string'
                 ? new PluginKey(options.pluginKey)
                 : options.pluginKey,
-        view: (view) => new DotBubbleMenuPluginView({ view, ...options })
+        view: (view) => new DotBubbleMenuPluginView({ view, ...options }),
+        props: {
+            /**
+             * Catch and handle the keydown in the plugin
+             *
+             * @param {EditorView} view
+             * @param {KeyboardEvent} event
+             * @return {*}
+             */
+            handleKeyDown(view: EditorView, event: KeyboardEvent) {
+                const { key } = event;
+                if (component.dropdown.showSuggestions) {
+                    if (key === 'Escape') {
+                        component.dropdown.toggleSuggestions();
+                        return true;
+                    }
+
+                    if (key === 'Enter') {
+                        component.dropdown.execCommand();
+                        return true;
+                    }
+
+                    if (key === 'ArrowDown' || key === 'ArrowUp') {
+                        component.dropdown.updateSelection(event);
+                        return true;
+                    }
+                }
```

Comment:
```suggestion
                if (component.dropdown.showSuggestions) {
                    if (key === 'Escape') {
                        component.dropdown.toggleSuggestions();
                    }

                    if (key === 'Enter') {
                        component.dropdown.execCommand();
                    }

                    if (key === 'ArrowDown' || key === 'ArrowUp') {
                        component.dropdown.updateSelection(event);
                    }

                    return true;
                }
```

Reply:
No because, if the suggestion is open wee should keep tracking other keys in the editor.

Reply:
yes we do.